src/sailsSocket.js

```diff
@@ -190,32 +200,32 @@ angular.module('bethel.sailsSocket', []).provider('sailsSocket', function() {
         //   {model: "task", verb: "destroyed", id: 20}
         socket.on(model, function(message) {
           switch (message.verb) {
-
-          case 'created':
-            if (findIndexById(scope, message.id) !== null) return;
-            scope.unshift(message.data);
-            break;
-
-          case 'destroyed':
-            const deleteIndex = findIndexById(scope, message.id);
-            if (deleteIndex !== null) {
-              scope.splice(deleteIndex, 1);
+            case 'created':
+              if (findIndexById(scope, message.id) !== null) return;
+              scope.unshift(message.data);
+              break;
+
+            case 'destroyed': {
```

Comment:
This is actually interesting, didn't know about it http://eslint.org/docs/rules/no-case-declarations